### PR TITLE
[BLOG] Add introducing-versioned-hdf5

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-introducing-versioned-hdf5-quansight.vercel.app/blog/introducing-versioned-hdf5).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by noatamir** [comment]
The feature image is cropped a bit awkwardly in the bottom. Can you re-crop or resize the size for the feature, or remove the text at the bottom to avoid this?
The crop at the top is not ideal either, but less disturbing in my opinion.
A screenshot of the feature image with an orange box highlighting the croped text at the bottom of the image

**Comment by noatamir** [comment]
Suggested description:

> HDF5 is an open technology that implements a hierarchical structure (similar to a file-system structure) for storing large amounts of possibly heterogeneous data within a single binary file, using regular version control tools (such as git) may prove difficult. The Versioned HDF5 library is a versioned abstraction on top of h5py, that allows you to keep a record of which changes occurred to your HDF5 files, and enables you to recover previous versions of this file.

**Comment by noatamir** [comment]
@gabalafou CC @trallard 
I noticed that the code block render something differently in the old blog and the new one. I am uncertain whether it requires a change or not. Can you have a look at these two images?
New code block:
Screenshot of new code block, highlighting change in rendered code, single dot
Old code block, [link to original blog](https://labs.quansight.org/blog/2020/08/introducing-versioned-hdf5):
Screenshot of new code block, highlighting change in rendered code, ellipses

**Comment by MarsBarLee** [comment]
@noatamir Thank you for the review! I changed the description to your suggestion and updated the feature image so it wouldn't be cropped anymore.

File: apps/labs/posts/introducing-versioned-hdf5.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+title: 'Introducing Versioned HDF5'
+published: August 21, 2020
+author: melissa-mendonca
+description: 'HDF5 is an open technology that implements a hierarchical structure (similar to a file-system structure) for storing large amounts of possibly heterogeneous data within a single binary file, using regular version control tools (such as git) may prove difficult. The Versioned HDF5 library is a versioned abstraction on top of h5py, that allows you to keep a record of which changes occurred to your HDF5 files, and enables you to recover previous versions of this file.'
+category: [PyData Ecosystem]
```
Comment (noatamir) [inline apps/labs/posts/introducing-versioned-hdf5.md:6]:
```suggestion
category: [PyData ecosystem]
```

**Review by noatamir — APPROVED** [review]
LGTM. merging

**Pull request merged**